spoj_practice/spoj2/spoj2.py

```diff
@@ -51,12 +38,14 @@ def find_primes_between (n, m):
     # m = numbers[0]
     # n = numbers[1]
     # instead, do:
-    m, n = stdin.readline().split(' ')
-    m = int(m)
-    n = int(n)
+    m, n = map(int, stdin.readline().split(' '))
     prime = []
```

Comment:
1) TIL. is what i just described a C++ nuance?. so every time an object is declared in a python, it's defined as a completely new object? there's no "overwriting" happening?

2) also true.